docs/REST API/rest-api.md

```diff
@@ -3826,6 +3828,92 @@ Retrieves a list of the configuration profiles assigned to a host.
 }
 ```
 
+### Live query one host (ad-hoc)
+
+Runs an ad-hoc live query against the specified host and responds with the results.
+
+The live query will stop if the targeted host is offline, or hasn't responded after 25 seconds (or whatever time period is configured via environment variable, e.g. `FLEET_LIVE_QUERY_REST_PERIOD=90s`).
+
+
+`POST /api/v1/fleet/hosts/:id/query`
+
+#### Parameters
+
+| Name      | Type  | In   | Description                                                                                                                                                        |
+|-----------|-------|------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| identifier       | integer  | path | **Required**. The target host ID. |
+| query            | string   | body | **Required**. The query SQL. |
+
+
+#### Example
+
+`POST /api/v1/fleet/hosts/123/query`
+
+##### Request body
+
+```json
+{
+  "query": "SELECT model, vendor FROM usb_devices;"
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "host_id": 123,
+  "query": "SELECT model, vendor FROM usb_devices;",
+  "status": "offline",
+  "error": null,
+  "rows": []
+}
+```
+
+### Live query host by identifier (ad-hoc)
+
+Runs an ad-hoc live query against a host identified using `uuid` and responds with the results.
+
+The live query will stop if the targeted host is offline, or hasn't responded after 25 seconds (or whatever time period is configured via environment variable, e.g. `FLEET_LIVE_QUERY_REST_PERIOD=90s`).
```

Comment:
Let's document that this could happen if the `distributed_interval` seconds in the agent settings is set higher than the configured `FLEET_LIVE_QUERY_REST_PERIOD` (defaults values are 10s and 25s respectively).

Reply:
@lucasmrod guessing that's also the case for the existing live query endpoint? https://fleetdm.com/docs/rest-api/rest-api#run-live-query

Reply:
Maybe:

```suggestion
The live query will stop if the targeted host is offline, or the query times out. Timeouts happen if the host hasn't responded after the configured `FLEET_LIVE_QUERY_REST_PERIOD` (default 25 seconds) or if the `distributed_interval` agent option (default 10 seconds) is higher than the `FLEET_LIVE_QUERY_REST_PERIOD`.
```
(feels kinda wordy though; I wonder if there's somewhere we can document this info for all 3 endpoints)

Reply:
Looks good!